tutorials/model_command.md

```diff
@@ -0,0 +1,271 @@
+\page model_command Model Command
+
+## Overview
+This command allows you to get the following information about models in the running world:
+
+- Model name.
+- Model pose.
+- Model links.
+  - Link name.
+  - Link parent.
+  - Link mass.
+  - Link inertial matrix.
+  - Link pose.
+- Model joints.
+  - Joint name.
+  - Joint parent.
+  - Joint parent link.
+  - Joint child link.
+
+### Available options for the model command
+
+```
+  --list                     Get a list of the available models.        
+  -m [--model] arg           Select the model to be shown.              
+  -l [--link]  arg           Select a link to show its properties.      
+                             If no arg is passed all links are printed  
+                             Requires the -m option                     
+                                                                        
+  -j [--joint] arg           Select a joint to show its properties.     
+                             If no arg is passed all joints are printed 
+                             Requires the -m option                     
+
+  -h [ --help ]              Print the help message.
+```
+
+
+## Example running the diff_drive world
+
+After having an Ignition Gazebo server running with the diff_drive world, you may check the available models running `ign model --list`
+
+<details>
+  <summary>Click to see output of `ign model --list`</summary>
+
+```
+      Requesting state for world [diff_drive] on service [/world/diff_drive/state]...
```

Comment:
Probably printing which service internally we are calling isn't accurate. However, we can definitely print the name of the world/simulation from where the models are being obtained.

Reply:
So this message would be better as Requesting state for world [diff_drive] in this case. Would this apply for only the tutorial or also in the implementation?

Reply:
Addressed